.evergreen/config.yml

```diff
@@ -166,7 +166,7 @@ functions:
           . ${DRIVERS_TOOLS}/.evergreen/venv-utils.sh
           . ${DRIVERS_TOOLS}/.evergreen/find-python3.sh
 
-          export PYTHON3_BINARY="$(find_python3)"
+          export PYTHON3_BINARY="$(find_python3 2>/dev/null)"
```

Comment:
What is the reason for discarding error messages when running find_python3?

Reply:
That's the recommended usage pattern, it prints debug info to stderr.